floppy/01-install-wget.cmd

```diff
@@ -23,10 +23,17 @@ if exist "%filename%" goto exit0
 echo ==^> Downloading "%WGET_URL%" to "%filename%"
 
 :powershell
+if defined http_proxy (
+    powershell -Command "$wc = (New-Object System.Net.WebClient); $wc.proxy = (new-object System.Net.WebProxy('%http_proxy%')) ; $wc.proxy.BypassList = (('%no_proxy%').split(',')) ; $wc.DownloadFile('%WGET_URL%', '%filename%')" >nul
```

Comment:
This configuration works if you download a file over HTTP but not over HTTPS. It fails with the first download of wget with my proxy and it works if I use `%https_proxy%`.